### DISCUSSION TITLE
Ensoniq EPS16plus sampler

### DISCUSSION THREAD

**Discussion by Johnnyd1965** [erichelgeson/BlueSCSI-v1#123] [General]
I tried the the Bluescsi with an ensoniq EPS16plus sampler.  I created an a few images using ensoniqFS and Total commander.    Can also be done with Chicken systems translator.   I created a blank image, one with the operating system via chicken system translator and also created an image of my physical hard drive.     The ensoniq can mount the images.   I am able to read from the drive via the sampler.  I can also load the operating system from the bluescsi.  The only problem I have is I can’t successfully write to the bluescsi I via the sampler.     I keep getting an error.   When I view the files on the computer via ensoniqfs And total commander, they are there but all corrupted.   Also am unable to format the scsi drives via the sampler.    Am I doing something wrong?

**Comment by Eugene1973-youngfather** [comment]
Hello. I join the question. I have an Ensoniq asr-10 sampler and it also cannot record and format discs Bluescsi and Bluescsi-F4, although both of these functions work fine with Ardscsino v2. What needs to be changed in the code?

**Comment by SirKrisBo** [comment]
I have both the ASR-10 and EPS 16+. Same issue for me. BlueSCSI is useful for loading sounds from CD images, but I've been unable to format or write to image files. It works perfectly with Akai samples though.

**Comment by Johnnyd1965** [comment]
I have tried everything.  I can only read from the Bluescsi on the Ensoniq Eps16.   The Bluescsi works well on the my Akai and Emu samplers.  

 I bought a Zuluscsi and it works well with the Ensoniq.   The only issue I initially had with the zuluscsi is if I named any CD images as CD1.img or CD1.iso they were not recognised.   I needed to name them as HD1.hda and they work well.

**Reply by SirKrisBo** [reply]
Did you buy the ZuluSCSI from Bruce? I think he might have told me about you. I might need to get one for my EPS 16. I'm currently using SCSI2SD with my ASR-10 but I'm keen to be able to also load sound from my CD images.
